packages/sdk/src/actions/listToken.ts

```diff
@@ -58,11 +60,13 @@ export async function listToken(
     listings.forEach((listing) => {
       if (
         (!listing.orderbook || listing.orderbook === 'reservoir') &&
-        client.marketplaceFees &&
         !('fees' in listing) &&
         !('marketplaceFees' in listing)
       ) {
-        listing.marketplaceFees = client.marketplaceFees
+        listing.marketplaceFees =
```

Comment:
Had some issues in the past setting api fields as undefined, it's probably better if we don't set it at all. You can do that but just have an if, else if check

Reply:
👍